docs/mmds/mmds-user-guide.md

```diff
@@ -0,0 +1,250 @@
+# microVM Metadata Service
+
+The Firecracker microVM Metadata Service (MMDS) is a mutable data store which
+can be used for sharing information between host and guests, in a secure and
+easy at hand way.
+
+# Activating the microVM Metadata Service
+
+By default, MMDS is not reachable from the guest operating system. At microVM
+runtime, MMDS is tightly coupled with a network interface, which allows MMDS
+requests. When configuring the microVM, if MMDS needs to be activated, a
+network interface has to be configured to allow MMDS requests. Network
+interface configuration API can be found in the
+[firecracker swagger file](../../src/api_server/swagger/firecracker.yaml).
+
+### Example
+
+```bash
+MMDS_NET_IF=eth0
+curl --unix-socket /tmp/firecracker.socket -i \
```

Comment:
nit: align the `\` characters at the same column

Reply:
Done.